apps/src/aichat/views/ChatEventView.tsx

```diff
@@ -64,12 +67,22 @@ const ChatItemView: React.FunctionComponent<ChatItemViewProps> = ({item}) => {
     );
   }
 
-  if (isModelUpdate(item)) {
+  if (isModelUpdate(event)) {
     return (
       <Alert
-        text={formatModelUpdateText(item)}
+        text={formatModelUpdateText(event)}
+        type="success"
+        onClose={() => dispatch(removeUpdateMessage(event.id))}
+        size="s"
+      />
+    );
+  }
+
+  if (event.descriptionKey) {
+    return (
+      <Alert
```

Comment:
I'm sure we'll iterate on this when we actually start displaying chat history events - I was thinking that we might want to display the non-alert events (like clear/load) differently from notifications? Probably something we can run by Mark/Sam in a later PR

Reply:
Sounds good - when I get to point of fetching student chat history on the frontend - will definitely consult with Mark/Sam. Thanks!